data/mods/Dark-Skies-Above/monattack.json

```diff
@@ -0,0 +1,46 @@
+[
+  {
+    "id": "pneumatic_bite",
+    "type": "monster_attack",
+    "attack_type": "bite",
+    "cooldown": 40,
+    "accuracy": 6,
+    "effects": [ { "id": "grabbed", "duration": 1000, "bp": "torso" } ],
+    "body_parts": [ [ "arm_l", 3 ], [ "arm_r", 3 ] ],
+    "no_infection_chance": 100,
+    "damage_max_instance": [
+      { "damage_type": "stab", "amount": 9, "armor_multiplier": 0.5 },
+      { "damage_type": "bash", "amount": 15, "armor_multiplier": 0.5 }
+    ],
+    "hit_dmg_u": "The %1$s grabs your arm and bites down hard!!",
+    "hit_dmg_npc": "The %1$s grabs <npcname> and bites down hard!!",
+    "no_dmg_msg_u": "The %1$s grabs your arm in its mouth, but fails to penetrate your armor.",
+    "no_dmg_msg_npc": "The %1$s grabs <npcname>, but fails to penetrate the armor."
+  },
+  {
+    "type": "monster_attack",
+    "attack_type": "melee",
+    "id": "tool_slash",
+    "cooldown": 20,
+    "move_cost": 150,
+    "damage_max_instance": [ { "damage_type": "cut", "amount": 8 } ],
+    "body_parts": [
+      [ "foot_l", 2 ],
+      [ "foot_r", 2 ],
+      [ "leg_l", 3 ],
+      [ "leg_r", 3 ],
+      [ "hand_l", 2 ],
+      [ "hand_r", 2 ],
+      [ "head", 3 ],
+      [ "eyes", 2 ],
+      [ "mouth", 1 ],
+      [ "arm_l", 3 ],
+      [ "arm_r", 3 ],
+      [ "torso", 4 ]
+    ],
+    "hit_dmg_u": "The %1$s slashes you with an integrated tool!",
+    "hit_dmg_npc": "The %1$s slashes <npcname>!",
+    "no_dmg_msg_u": "The %1$s attempts to cut you, but fails to penetrate your armor.",
+    "no_dmg_msg_npc": "The %1$s tries to cut <npcname>, but fails to penetrate your armor."
```

Comment:
your -> their

Reply:
amusingly I saw this earlier, thought "ah I'll make sure I fix that!" and then promptly forgot to do so. thanks for the heads up!